packages/webapp/src/containers/Finances/CustomRevenueType/ReadonlyCustomRevenue.jsx

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import PureSimpleCustomType from '../../../components/Forms/SimpleCustomType';
+import { HookFormPersistProvider } from '../../hooks/useHookFormPersist/HookFormPersistProvider';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { retireCustomRevenueType } from '../saga';
```

Comment:
Unfortunately I called this one `deleteRevenueType` 😆 Feel free to enforce a consistent naming between expense and revenue types instead!! But as is, this will crash on import.

Reply:
Yeah I am considering that for just this one. I like delete because it matches others and the request. But I like retire because its what we use in the app and it is possible to unretire. I might rename just this one..

Reply:
then I got to rename deleteRevenueTypesSuccess -- I am just going to leave it  😆